FEM/pcs_dm.cpp

```diff
@@ -772,19 +772,21 @@ double CRFProcessDeformation::Execute(int loop_process_number)
 
 				Error = Norm / InitialNorm;
 				ErrorU = NormU / InitialNormU0;
-				if (Norm < Tolerance_global_Newton && Error > Norm)
-					Error = Norm;
+				// Error = Norm / InitialNorm;
```

Comment:
It these lines are not needed, you can remove them. Also for some new commented lines below.